src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -333,33 +337,38 @@ private IList<string> GetConfigFilePaths(ISettings settings)
             return settings.GetConfigFilePaths();
         }
 
-        private static PackageReference[] GetPackageReferences(PackageSpec packageSpec)
+        private async Task<IEnumerable<PackageReference>> GetPackageReferences(PackageSpec packageSpec)
```

Comment:
Since it's an async method, can you add **Async suffix to the method name?
```
        private async Task<IEnumerable<PackageReference>> GetPackageReferencesAsync(PackageSpec packageSpec)
```